metrics/metrics.go

```diff
@@ -151,5 +151,7 @@ func RegisterMetrics() {
 	prometheus.MustRegister(TiKVRangeTaskStats)
 	prometheus.MustRegister(TiKVRangeTaskPushDuration)
 	prometheus.MustRegister(HandleSchemaValidate)
+	prometheus.MustRegister(TiKVTokenWaitDuration)
```

Comment:
Is this metric used? Why is it picked into this PR?

Reply:
Done @MyonKeminta